portal-web/test/functional/com/liferay/portalweb/macros/FormPortlet.macro

```diff
@@ -77,6 +77,22 @@ definition {
 		Alert.viewSuccessMessage();
 	}
 
+	macro submitSuccessfullyAndViewSuccessPage {
+		FormPortlet.submitSuccessfully();
+
+		if (!(isSet(successPageDescription))) {
+			var successPageDescription = "Your information was successfully received. Thank you for filling out the form.";
+		}
+
+		if (!(isSet(successPageTitle))) {
+			var successPageTitle = "Thank you.";
+		}
+
+		FormPortlet.viewSuccessPage(
+			successPageDescription = "${successPageDescription}",
+			successPageTitle = "${successPageTitle}");
+	}
+
```

Comment:
Hi @leilanyaragao , this macro is already in master, could you please remove your commit?